supplements/tashas-cauldron-of-everything/archetypes/druid-wildfire.xml

```diff
@@ -151,7 +139,7 @@
 			<set name="size">Small</set>
 			<set name="senses">darkvision 60 ft., passive Perception 12</set>
 			<set name="languages">understands the languages you speak</set>
-			<set name="alignment">any chaotic alignment</set>
+			<set name="alignment"></set>

```

Comment:
changing it like this, would mean that it would read
> `Small elemental,`

since it sees empty space as still being an alignment. And since we can't delete that set, maybe we just put "unaligned" there, yes, isn't exactly like in the book, but it would mean the same thing, and there wouldn't be any useless commas.